cli/install.go

```diff
@@ -205,8 +205,62 @@ type githubTaggedReleaseResponse struct {
 }
 
 type gitHubAsset struct {
-	Url  string // download url for asset
-	Name string // contains platform information about binary
+	Url                  string // url for asset json object
+	Name                 string // contains platform information about binary
+	Browser_download_url string // download url
+}
+
+func GetZlsDownloadUrl(version string, archDouble string) (string, error) {
+	if version == "master" {
+		var exeName = "zls"
+		if strings.Contains(archDouble, "windows") {
+			exeName = "zls.exe"
+		}
+		return fmt.Sprintf("https://zig.pm/zls/downloads/%v/bin/%v", archDouble, exeName), nil
+
+	} else {
+		var releaseUrl string
+		if version == "master" {
+			releaseUrl = "https://api.github.com/repos/zigtools/zls/releases/latest"
```

Comment:
isnt this never going to be reached because of the first if?

Reply:
You're absolutely right! It's a dead branch. I left it in unintentionally, but perhaps it might be desirable to leave it in. When the language and ZLS itself mature I presume the latest releases from the ZLS repo are going to be much more desirable than the ones from the https://zig.pm/ resource.

Reply:
Thanks! Fixed in new draft